WorkloadManagementSystem/DB/JobElasticDB.py

```diff
@@ -0,0 +1,142 @@
+""" DIRAC JobDB class is a front-end to the main WMS database containing
+  job definitions and status information. It is used in most of the WMS
+  components
+
+  The following methods are provided for public usage:
+
+  getJobParameters()
+
+  setJobParameter()
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC.Core.Utilities.ReturnValues import S_OK, S_ERROR
+from DIRAC.Core.Base.ElasticDB import ElasticDB as DB
+from DIRAC.ResourceStatusSystem.Client.SiteStatus import SiteStatus
+from DIRAC import gLogger
+
+#############################################################################
+
+JOB_STATES = ['Submitting', 'Received', 'Checking', 'Staging', 'Waiting', 'Matched',
+              'Running', 'Stalled', 'Done', 'Completed', 'Failed']
+JOB_FINAL_STATES = ['Done', 'Completed', 'Failed']
+
```

Comment:
The 2 above are a copy-paste from JobDB.py, and they actually seem useless here.

Reply:
I agree. Removed it from the code.